utilities/Tools.js

```diff
@@ -1,5 +1,5 @@
 const { EluvioLiveStream } = require("../src/LiveStream.js");
-const { Config } = require("../src/Config.js");
```

Comment:
This file is committed by mistake - let's remove so we don't have to make this fix

Reply:
Should the file be renames from `Config.js` to `Environments.js`?  I think it is useful to have this defined.  The key issue is the first line of that files:

```js
const net = "main"; // Set to "main"  "demov3" "test" "dev"
```

Since this is the only place it is defined without my proposed changes.  And a user can still use this without ``-network` and it will use `main`